yamscripts.yml

```diff
@@ -19,7 +19,7 @@ scripts:
     pre: =>clean && =>copyTestImages
     static: cpx "content-src/static/**/*" data/content
   # TODO: make the number of locales configurable. pontoon-to-json has an option for this
-    locales: cpx "locales/**/*" data/locales
+    locales: pontoon-to-json --src locales --dest data/locales
```

Comment:
`pontoon-to-json` generates a single `json` with all `SUPPORTED_LOCALES`, so I suppose one approach is at add-on build time, for each locale, package up the locale with "en-US" as the default, e.g,. "fr" + "en-US" in `fr.json`. This way when building "fr" Firefox, it'll just package `fr.json` and ignore all the other locale files. A slight tweak on this is we could have a smarter `fr.json` at add-on build time by detecting if strings are missing and selectively use "en-US" strings, so there's only one "locale" in a given `json` file, but it's potentially a mix of translated and default strings.

But on the flip side of just packaging all locales and all strings... there's currently ~30 strings and `loop-client-l10n` had ~80 locales, so that's 2400 strings in a combined `locales.json`.